packages/mux-video/src/CustomVideoElement.js

```diff
@@ -218,33 +239,38 @@ for (
   });
 }
 
-// Passthrough native el functions from the custom el to the native el
-nativeElProps.forEach((prop) => {
-  const type = typeof nativeElTest[prop];
-
-  if (type == 'function') {
-    // Function
-    CustomVideoElement.prototype[prop] = function () {
-      return this.nativeEl[prop].apply(this.nativeEl, arguments);
-    };
-  } else {
-    // Getter
-    let config = {
-      get() {
-        return this.nativeEl[prop];
-      },
-    };
-
-    if (prop !== prop.toUpperCase()) {
-      // Setter (not a CONSTANT)
-      config.set = function (val) {
-        this.nativeEl[prop] = val;
+forwardProps(nativeElTest, nativeElProps);
+
+function forwardProps(nativeEl, props) {
```

Comment:
non-blocking: this will handle properties (with the concerns mentioned above). What about attributes? Should we e.g. check the `observedAttributes` of the element to ensure proper forwarding?

Reply:
good call, will check this out.